www/src/graph/installations.graphql

```diff
@@ -18,31 +29,3 @@ fragment Installation on Installation {
     ...OIDCProvider
   }
 }
-
-query GetInstallation($name: String) {
```

Comment:
is it ok these are all removed?

Reply:
Yup, I verified that they weren't used anywhere.

Reply:
kk